class/task.php

```diff
@@ -56,9 +56,9 @@ function show_message( $message ) {
                     'title'                    => __( 'Title', 'cpm' ),
                     'start_date'               => __( 'Start Date', 'cpm' ),
                     'due_date'                 => __( 'Due Date', 'cpm' ),
-                    'to_attach'                => __( 'To attach', 'cpm' ),
-                    'select_file'              => __( 'select files', 'cpm' ),
-                    'from_computer'            => __( 'from your computer', 'cpm' ),
+                    //'to_attach'                => __( 'To attach', 'cpm' ),
+                    //'select_file'              => __( 'select files', 'cpm' ),
+                    //'from_computer'            => __( 'from your computer', 'cpm' ),

```

Comment:
Yes I can.
I've commented and not deleted because I thought you might need it somewere else.
But I think it's not usefull anymore, because those were concatenated in a single string as you can see here:
https://github.com/weDevsOfficial/wp-project-manager/pull/119/commits/c5cf4f254037dec5c480f41705049d2ac935cf22